docs/products/postgresql/howto/pagila.rst

```diff
@@ -0,0 +1,251 @@
+
+Sakila (and thus, Pagila) is a database representing a DVD rental store (remember those?!), containing information about films (like title, category, actresses), rental stores (like address, staff members, customers) and rentals, where a customer rents a film from a store through its staff.
+
+With all these relational information, Pagila is a perfect fit to play around with PostgreSQL and the SQL language.
+
+Load Pagila on PostgreSQL
+-------------------------
+
+Before exploring the Pagila database, we need a PostgreSQL server to load the data on. A quick and easy way to get a server is through `Aiven for PostgreSQL <https://aiven.io/postgresql>`_. If don't have an Aiven account yet, `sign up <https://console.aiven.io/signup?utm_source=github&amp;utm_medium=organic&amp;utm_campaign=devportal&amp;utm_content=repo>`_ and enjoy our free trial!
+
+After logging in the Aiven Console, chose the PostgreSQL version, select the cloud and plan of your choice, give the service a name and hit the "Create Service" button. In a couple of minutes, you will have a PostgreSQL server up and running.
+
+.. image:: /images/products/postgresql/pg-connection-details.png
+    :alt: PostgreSQL service overview tab in Aiven's console
+
+Now, we will load the Pagila database into our new service.
+
+1. Download the ``pagila-data.sql`` from our `GitHub repository <https://github.com/aiven/devportal/blob/paglia-sample-database/code/products/postgresql/pagila/pagila-data.sql>`_.
+
+.. Tip::
+    You may use the following command on your terminal::
+
+        wget https://raw.githubusercontent.com/aiven/devportal/paglia-sample-database/code/products/postgresql/pagila/pagila-data.sql
+
+2. Connect to the PostgreSQL instance using the command below. The ``SERVICE_URI`` value can be found in the Aiven Console dashboard.
+
+.. code:: shell
+
+   psql 'SERVICE_URI'
+
+3. Within the ``psql`` shell, create a database named ``pagila`` and connect to it with the command below:
+
+.. code:: psql
+
+    CREATE DATABASE pagila;
+    \c pagila;
+
+4. Populate the database with the following below. This might take some time, so grab a beverage of your choice ;)
+
+.. code:: psql
+
+    \i pagila-data.sql
+
+5. You are ready to go! You can use the `Sample queries` section below to explore the database. Have fun!
+
+Entity-relationship model diagram
+---------------------------------
+
+The image below shows an overview of the Pagila database tables and views, generated by `DBeaver <https://dbeaver.io>`_.
```

Comment:
Is this part of the generic info for this sample data or is it how we've set it up? If it's specific to Aiven, I would move this up to where the conceptual info sits (what is Pagila etc.), and maybe even think of separating the conceptual info from the actual task.

Reply:
This is info for the sample data, not related to Aiven itself, it only makes sense on this article.